src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -58,6 +58,7 @@ export interface IMultipleCategoryBarPlotProps {
     svgRef?: (svgContainer: SVGElement | null) => void;
     pValue: number | null;
     qValue: number | null;
+    sortOption: string;
```

Comment:
we can add a question mark like `sortOption?: string;` to make this prop optional. then if this prop is not provided, we can assume it to be 
'sortByAlphabet'. that way we don't have to pass `sortOption={'sortByAlphabet'}` every time